pkg/resources/cloudcontroller/kubevirt.go

```diff
@@ -32,8 +32,10 @@ import (
 )
 
 const (
-	KubeVirtCCMDeploymentName = "kubevirt-cloud-controller-manager"
-	KubeVirtCCMTag            = "v0.1.0"
+	KubeVirtCCMDeploymentName                       = "kubevirt-cloud-controller-manager"
+	KubeVirtCCMTag                                  = "v0.2.0"
+	CloudControllerManagerInfraKubeconfigSecretName = "cloud-controller-manager-infra-kubeconfig"
```

Comment:
Could you change this name? It's very long. 
Please change
`CloudControllerManagerInfraKubeconfigSecretName` to `KubeVirtInfraSecretName`
`InfraKubeConfigSecretKey` to `KubeVirtInfraSecretKey`